lib/mobility-core/src/Kernel/External/Maps/Interface/Types.hs

```diff
@@ -83,7 +83,8 @@ type GetDistancesResp a b = NonEmpty (GetDistanceResp a b)
 data GetRoutesReq = GetRoutesReq
   { waypoints :: NonEmpty LatLong,
     mode :: Maybe TravelMode, -- Defaults to CAR
-    calcPoints :: Bool -- True (default) if points needs to be calculated
+    calcPoints :: Bool, -- True (default) if points needs to be calculated
+    isAvoidToll :: Bool
```

Comment:
It can cause backward incompatibility issues. As this API is being called directly from the UI. Its better not to make it part of request, instead of it passing as a separate parameters inside getRoutes function.

Reply:
got it.